app/src/test/java/com/kickstarter/viewmodels/PledgeFragmentViewModelTest.kt

```diff
@@ -967,7 +965,10 @@ class PledgeFragmentViewModelTest : KSRobolectricTestCase() {
         this.increasePledgeButtonIsEnabled.assertValuesAndClear(true, false)
         this.additionalPledgeAmountIsGone.assertValuesAndClear(true, false)
         this.additionalPledgeAmount.assertValuesAndClear("$0", "$1")
+    }
 
+    @Test
+    fun testPledgeStepping_almostMaxReward_MXProject() {
```

Comment:
Only thing I would add is a behavior test for checking that the animation fired on backpress 👍